source/includes/tutorial1/_overview.html.md

```diff
@@ -0,0 +1,65 @@
+# Problem Overview 
+
+> ![RL Overview Image](../images/tutorial1-rl-overview.png)
+
+In this tutorial you will learn about reinforcement learning (RL), and using RL to solve a simple simulated problem using Inkling and the Bonsai Platform. This will include learning the essential components of the Inkling language needed to connect the simulation to the platform.  You will familiarize yourself with the simulation, test it, then fill in the necessary Inkling schemas (states and actions) needed to connect the simulation to the platform. By the end of this tutorial, you can expect to be comfortable with the relationship between a BRAIN and a simulation, as well as how to train a BRAIN inside the Bonsai Platform using custom Inkling code, and basic knowledge of Bonsai’s Command Line Interface (CLI).
+
+We expect you to already have read the [Quick Start tutorial overview][1] and [install the CLI][2] before following this tutorial so you have a general understanding of how the platform works and have installed and configured the Bonsai CLI first. Please do those two things before continuing with this tutorial.
```

Comment:
remove final "first" from first sentence. Could probably also remove the second sentence -- it's redundant.